freqtrade/strategy/strategy.py

```diff
@@ -69,7 +70,9 @@ def init(self, config: dict) -> None:
             )
 
         # Minimal ROI designed for the strategy
-        self.minimal_roi = self.custom_strategy.minimal_roi
+        self.minimal_roi = OrderedDict(sorted(
```

Comment:
It will not be better to store this result and return it when we call strategy.minimal_roi instead of sorting it again?

Reply:
Well as long as this is always the place where we set the table, this is the simplest solution. When we have something else, then maybe we need a getter with a cache. Now I think we are fine with this.